frontend/src/components/ProductColumn/ProductColumn.tsx

```diff
@@ -0,0 +1,42 @@
+import type { Product } from "@/types/Product.types";
+
+import { Card } from "../Card";
+import { Text } from "../retroui";
+
+type ProductColumnProps = {
+  data: Product[];
+  store: string;
+};
+
+export function ProductColumn({ data, store }: ProductColumnProps) {
+  return (
+    <div
+      data-testid="nw"
+      className={`supermarket-container w-full pb-10 ${store === "Pak'nSave" ? "bg-gray-100" : "bg-gray-300"}`}
+    >
+      <Text as={"h2"} className="py-6 text-center">
+        {store}
+      </Text>
```

Comment:
Should check `store` exists in order to render it and what does it display if it doesn't have the name?